libraries/src/Installer/Adapter/FileAdapter.php

```diff
@@ -172,7 +172,11 @@ protected function finaliseInstall()
      */
     protected function finaliseUninstall(): bool
     {
-        File::delete(JPATH_MANIFESTS . '/files/' . $this->extension->element . '.xml');
+        $manifest = JPATH_MANIFESTS . '/files/' . $this->extension->element . '.xml';
+
+        if (\is_file($manifest)) {
```

Comment:
```suggestion
        if (is_file($manifest)) {
```
At all other places you have used `is_file`, only here you've used `\is_file`. Both is ok, but it should be consistent.

Reply:
You are right. I just changed it...